.github/workflows/ubuntu.yml

```diff
@@ -0,0 +1,66 @@
+name: ubuntu
+
+on:
+  pull_request:
+
+  push:
+    branches:
+      - staging
+      - trying
+
+jobs:
+  ubuntu:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        ruby: [ 2.3.8, 2.4.9, 2.5.7, 2.6.5, head ]
+        rgv: [ v2.5.2, v2.6.14, v2.7.10, v3.0.6, master ]
+        exclude:
+          - ruby: 2.4.9
+            rgv: v2.5.2
+          - ruby: 2.5.7
+            rgv: v2.5.2
+          - ruby: 2.5.7
+            rgv: v2.6.14
+          - ruby: 2.6.5
+            rgv: v2.5.2
+          - ruby: 2.6.5
+            rgv: v2.6.14
+          - ruby: 2.6.5
+            rgv: v2.7.10
+          - ruby: head
+            rgv: v2.5.2
+          - ruby: head
+            rgv: v2.6.14
+          - ruby: head
+            rgv: v2.7.10
+          - ruby: head
+            rgv: v3.0.6
+    steps:
+      - uses: actions/checkout@v1
+      - name: Setup ruby
+        uses: ruby/setup-ruby@v1
+        with:
+          ruby-version: ${{ matrix.ruby }}
+      - name: Prepare dependencies
+        run: |
+          sudo apt-get install graphviz -y
+          bin/rake override_version
+          bin/rake spec:parallel_deps
+      - name: Run Test
+        run: bin/parallel_rspec spec
+        env:
+          RGV: ${{ matrix.rgv }}
```

Comment:
Similarly to the comment in the bundler3 workflow, this environment variable probably makes sense at the "job level"?